src/libs/elektra/keytest.c

```diff
@@ -331,21 +346,24 @@ int keyIsDirectlyBelow (const Key * key, const Key * check)
 }
 
 /**
- * Check if a key is binary type.
+ * Check if the value of a @p key is of binary type.
  *
- * The function checks if the key is a binary. Opposed to string values binary
- * values can have '\\0' inside the value and may not be terminated by a null
- * character. Their disadvantage is that you need to pass their size.
+ * The function checks if the value of @p key is binary. Opposed to string
```

Comment:
`Contrary` / `As opposed` to
I think?
```suggestion
 * The function checks if the value of @p key is binary. Contrary to string
```